File: ProblemZR.java
public class ProblemZR{
    public static void main(String[] args) {

        System.out.println("This is leetcode Problem");
        
    }
    public ListNode deleteDuplicates(ListNode head) {
        
        ListNode  current = head;
        while ( current!= null && current.next != null ){
            if (current.val == current.next.val){
                current.next = current.next.next;
            }else{
                current = current.next;
            }
        
        }
        return head;
        
    }

    public class ListNode {
        int val;
        ListNode next;
        ListNode() {}
        ListNode(int val) { this.val = val; }
        ListNode(int val, ListNode next) { this.val = val; this.next = next; }
    }
}

File: ProblemZR.py
# Definition for singly-linked list.
# class ListNode(object):
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next
class Solution(object):
    def deleteDuplicates(self, head):
        """
        :type head: ListNode
        :rtype: ListNode
        """
        self.current = head 
        while(self.current and self.current.next):
            if(self.current.val == self.current.next.val):
                self.current.next = self.current.next.next;
            else:
                self.current = self.current.next;
        return head